src/components/oppfolgingsplan/lps/BehandleOppfolgingsplanLPS.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { Checkbox } from 'nav-frontend-skjema';
+import { toDatePrettyPrint } from '@navikt/digisyfo-npm';
+import { behandlePersonOppgave } from '../../../actions/personoppgave_actions';
+
+const behandleOppfolgingsplanLPSLabel = (erBehandlet, opLPSPersonOppgave) => {
+    return opLPSPersonOppgave && opLPSPersonOppgave.behandletTidspunkt
+        ? `Ferdigbehandlet av ${opLPSPersonOppgave.behandletVeilederIdent} ${toDatePrettyPrint(opLPSPersonOppgave.behandletTidspunkt)}`
+        : 'Marker som behandlet';
+};
+
+const BehandlOppfolgingsplanLPSPanel = styled.div`
+    margin-bottom: 1em;
+    border: 1px solid #C6C2BF;
+    .skjemaelement {
+        margin: 0;
+    }
+`;
+
+export const BehandleOppfolgingsplanLPS = (
+    {
+        oppfolgingsplanLPS,
+        veilederIdent,
```

Comment:
VeilederIdent brukes vel bare her? Er det mulig å hente den med en `useSelector` eller noe, i stedet for å sende den ned helt fra containeren?